xml/article_virtualization.xml

```diff
@@ -273,21 +274,21 @@
     Standard real-time dangers apply in that if the intention is to run a
     compute intensive application with realtime priority, then the user must
     ensure that kernel threads cannnot starve. A simple precaution is to use
-    <option>rtkthread=prio</option> and <option>rtworkqueues=prio</option>
-    kernel boot parameters, with priority set higher than anything that may
+    the <option>rtkthread=prio</option> and <option>rtworkqueues=prio</option>
+    kernel boot parameters, with priorities set higher than anything that may
     dominate a CPU. This is not strictly real-time capable, but it is safer
     overall.
    </para>
    <itemizedlist>
     <title>Docker Prerequisites</title>
     <listitem>
      <para>
-      kernel must be booted with <option>nortsched</option> command-line
+      The kernel must be booted with <option>nortsched</option> command-line
       parameter
      </para>
      <para>
       This is to hide cgroup scheduling from Docker. If cgroup scheduling is
-      required then isolating docker is very problematic.
+      required, then isolating Docker is very problematic.
```

Comment:
Docker's trademark policy wants us to use "Docker as an adjective" (by which they mean it must be used before a noun). In this case, it would also help clarify what we mean (engine/images/containers/...):

```suggestion
      required, then isolating Docker containers is very problematic.
```